scripts/fix_readability_bulk.sh

```
@@ -0,0 +1,75 @@
+    FIXES=$((FIXES + TRAILING_FIXES))
+    echo -e "${GREEN}✓ Fixed trailing whitespace in files${NC}"
+else
+    echo "⚠️  Could not fix trailing whitespace (files may already be clean)"
+fi
+
+echo -e "\n${YELLOW}2. Fixing single-function pipelines...${NC}"
+# Fix patterns like: data |> function() -> function(data)
+find lib -name "*.ex" -type f | while read -r file; do
+    if sed -i.bak -E 's/([[:alnum:]_]+)[[:space:]]+\|>[[:space:]]+([[:alnum:]_]+)\(\)/\2(\1)/g' "$file" 2>/dev/null; then
+        if ! cmp -s "$file" "$file.bak"; then
+            FIXES=$((FIXES + 1))
```

Shellcheck (0.10.0)

[info] Modification of FIXES is local (to subshell caused by pipeline).

(SC2030)